call-home/src/bin/callhome/main.rs

```diff
@@ -129,6 +138,19 @@ async fn generate_report(
         ..Default::default()
     };
 
+    let mut event_data = EventData::default();
+    match event_stats(transmission_url.clone()).await {
+        Ok(data) => {
+            event_data.volume_created = data.volume_created();
+            event_data.volume_deleted = data.volume_deleted();
+            event_data.pool_created = data.pool_created();
+            event_data.pool_deleted = data.pool_deleted();
+        }
+        Err(err) => {
+            error!("{:?}", err);
```

Comment:
Might want to qualify what went wrong specifically

Reply:
The same is just printed in this file for `match nodes` , `match pools`, ` match volumes` ., on the very same line its done in PR.